tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -799,11 +799,18 @@ isl::multi_union_pw_aff extractDomainToIds(
   for (auto mapping : tree->collect(tree, ScheduleTreeType::MappingFilter)) {
     auto mappingNode = mapping->elemAs<ScheduleTreeElemMappingFilter>();
     auto list = isl::union_pw_aff_list(tree->ctx_, ids.size());
+    bool hasIds = true;
     for (auto id : ids) {
-      CHECK_GT(mappingNode->mapping.count(id), 0) << "no mapping to id " << id;
+      if (mappingNode->mapping.count(id) == 0) {
+        hasIds = false;
+        break;
+      }
       auto idMap = mappingNode->mapping.at(id);
       list = list.add(idMap);
     }
+    if (!hasIds) {
+      continue;
+    }
```

Comment:
This changes the behavior, without modifying the documentation.
It also changes the behavior of extractMappingToThreads, which will
now silently ignore unmapped branches.